mirror.musicbrainz.recording.search/request

```diff
@@ -0,0 +1,2 @@
+GET https://ssl14.ovh.net/~hoodbrai/proxy.php?http://musicbrainz-mirror.eu:5000/ws/2/recording?fmt=json&query={{q}}
```

Comment:
Using a proxy is not mandatory. We don't enforce https between the cozy-stack and the remote service.